dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -104,13 +215,41 @@ DO
 		Dialog Dialog1
 		cancel_confirmation
 		Call validate_MAXIS_case_number(err_msg, "*")
-		IF verifs_needed = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
-		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
+		IF trim(verifs_needed) = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
+		IF len(trim(verifs_needed)) > 100 THEN err_msg = err_msg & vbCr & "* The list of verifications must be less than 100 characters long. Please shorten."
+		IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
 		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
 	LOOP UNTIL err_msg = ""									'loops until all errors are resolved
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 
+'Write information to SPEC/MEMO
+memo_header = "--- SNAP Closed " & closure_date & " - Expedited Autoclose ---"
+Call write_variable_in_SPEC_MEMO(memo_header)
+Call write_variable_in_SPEC_MEMO(" ")
+Call write_variable_in_SPEC_MEMO("Reason for closure: Delayed verifications were not submitted and Expedited SNAP Autoclosed.")
+Call write_variable_in_SPEC_MEMO(" ")
+If verifs_needed <> "" then call write_variable_in_SPEC_MEMO("Verifications needed: " & verifs_needed)
+Call write_variable_in_SPEC_MEMO(" ")
+CALL write_variable_in_SPEC_MEMO("*** Submitting Documents:")
+CALL write_variable_in_SPEC_MEMO("- Online at infokeep.hennepin.us or MNBenefits.mn.gov")
+CALL write_variable_in_SPEC_MEMO("  Use InfoKeep to upload documents directly to your case.")
+CALL write_variable_in_SPEC_MEMO("- Mail, Fax, or Drop Boxes at service centers listed below:")
+CALL write_variable_in_SPEC_MEMO(" - 7051 Brooklyn Blvd Brooklyn Center 55429")
+CALL write_variable_in_SPEC_MEMO(" - 1011 1st St S Hopkins 55343")
+CALL write_variable_in_SPEC_MEMO(" - 1001 Plymouth Ave N Minneapolis 55411")
+CALL write_variable_in_SPEC_MEMO(" - 2215 East Lake Street Minneapolis 55407")
+CALL write_variable_in_SPEC_MEMO(" (Hours are 8 - 4:30 Monday - Friday)")
+Call write_variable_in_SPEC_MEMO(" ")
+'To do - need to verify if this is necessary, won't it always include this info? 
```

Comment:
This can be removed.

Reply:
Removed this line of code related to SNAP REIN date